Build failure: fancontrol-gui-0.8 fails to build cause to a cmake version error

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Package fancontrol-gui-0.8 fails to build cause to a cmake version error
It seems to be this in the error logs : CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

### Can Hydra reproduce this build failure?

Yes

### Link to Hydra build job

https://hydra.nixos.org/build/310542464

### Relevant log output

```console
Running phase: configurePhase
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/doc/fancontroller -DCMAKE_INSTALL_INFODIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/include -DCMAKE_INSTALL_SBINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8 -DKDE_INSTALL_EXECROOTDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8 -DKDE_INSTALL_BINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/bin -DKDE_INSTALL_SBINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/sbin -DKDE_INSTALL_LIBDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib -DKDE_INSTALL_LIBEXECDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/libexec -DKDE_INSTALL_CMAKEPACKAGEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib/cmake -DKDE_INSTALL_INCLUDEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/include -DKDE_INSTALL_LOCALSTATEDIR=/var -DKDE_INSTALL_DATAROOTDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share -DKDE_INSTALL_DATADIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share -DKDE_INSTALL_DOCBUNDLEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/doc/HTML -DKDE_INSTALL_KCFGDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/config.kcfg -DKDE_INSTALL_KCONFUPDATEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/kconf_update -DKDE_INSTALL_KSERVICES5DIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/kservices5 -DKDE_INSTALL_KSERVICETYPES5DIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/kservicetypes5 -DKDE_INSTALL_KXMLGUI5DIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/kxmlgui5 -DKDE_INSTALL_KNOTIFY5RCDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/knotifications5 -DKDE_INSTALL_ICONDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/icons -DKDE_INSTALL_LOCALEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/locale -DKDE_INSTALL_SOUNDDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/sounds -DKDE_INSTALL_TEMPLATEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/templates -DKDE_INSTALL_WALLPAPERDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/wallpapers -DKDE_INSTALL_APPDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/applications -DKDE_INSTALL_DESKTOPDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/desktop-directories -DKDE_INSTALL_MIMEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/mime/packages -DKDE_INSTALL_METAINFODIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/metainfo -DKDE_INSTALL_MANDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/man -DKDE_INSTALL_INFODIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/info -DKDE_INSTALL_DBUSDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/dbus-1 -DKDE_INSTALL_DBUSINTERFACEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/dbus-1/interfaces -DKDE_INSTALL_DBUSSERVICEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/dbus-1/services -DKDE_INSTALL_DBUSSYSTEMSERVICEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/dbus-1/system-services -DKDE_INSTALL_SYSCONFDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/etc -DKDE_INSTALL_CONFDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/etc/xdg -DKDE_INSTALL_AUTOSTARTDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/etc/xdg/autostart -DKDE_INSTALL_QTPLUGINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib/qt-5.15.17/plugins -DKDE_INSTALL_PLUGINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib/qt-5.15.17/plugins -DKDE_INSTALL_QMLDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib/qt-5.15.17/qml -DECM_DIR=/nix/store/6qbd89v49lva9xkhxkjcgnjmzwd92kbg-extra-cmake-modules-5.116.0/share/ECM/cmake
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!
error: Cannot build '/nix/store/78ln9zb0by7qb1nkhkvbbvyh1kks6lh4-fancontrol-gui-0.8.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8
       Last 21 log lines:
       > Running phase: qtPreHook
       > Running phase: unpackPhase
       > unpacking source archive /nix/store/vlpfz3hpbhajpxvhzbafsp38a31mgzpj-source
       > source root is source
       > Running phase: patchPhase
       > applying patch /nix/store/zxgjn9hysiqm50x35fxlmvcz7633rj9d-version.patch
       > patching file fancontrol-gui/src/main.cpp
       > Running phase: updateAutotoolsGnuConfigScriptsPhase
       > Running phase: configurePhase
       > cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/doc/fancontroller -DCMAKE_INSTALL_INFODIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/include -DCMAKE_INSTALL_SBINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8 -DKDE_INSTALL_EXECROOTDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8 -DKDE_INSTALL_BINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/bin -DKDE_INSTALL_SBINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/sbin -DKDE_INSTALL_LIBDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib -DKDE_INSTALL_LIBEXECDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/libexec -DKDE_INSTALL_CMAKEPACKAGEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib/cmake -DKDE_INSTALL_INCLUDEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/include -DKDE_INSTALL_LOCALSTATEDIR=/var -DKDE_INSTALL_DATAROOTDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share -DKDE_INSTALL_DATADIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share -DKDE_INSTALL_DOCBUNDLEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/doc/HTML -DKDE_INSTALL_KCFGDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/config.kcfg -DKDE_INSTALL_KCONFUPDATEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/kconf_update -DKDE_INSTALL_KSERVICES5DIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/kservices5 -DKDE_INSTALL_KSERVICETYPES5DIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/kservicetypes5 -DKDE_INSTALL_KXMLGUI5DIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/kxmlgui5 -DKDE_INSTALL_KNOTIFY5RCDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/knotifications5 -DKDE_INSTALL_ICONDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/icons -DKDE_INSTALL_LOCALEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/locale -DKDE_INSTALL_SOUNDDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/sounds -DKDE_INSTALL_TEMPLATEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/templates -DKDE_INSTALL_WALLPAPERDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/wallpapers -DKDE_INSTALL_APPDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/applications -DKDE_INSTALL_DESKTOPDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/desktop-directories -DKDE_INSTALL_MIMEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/mime/packages -DKDE_INSTALL_METAINFODIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/metainfo -DKDE_INSTALL_MANDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/man -DKDE_INSTALL_INFODIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/info -DKDE_INSTALL_DBUSDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/dbus-1 -DKDE_INSTALL_DBUSINTERFACEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/dbus-1/interfaces -DKDE_INSTALL_DBUSSERVICEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/dbus-1/services -DKDE_INSTALL_DBUSSYSTEMSERVICEDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/share/dbus-1/system-services -DKDE_INSTALL_SYSCONFDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/etc -DKDE_INSTALL_CONFDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/etc/xdg -DKDE_INSTALL_AUTOSTARTDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/etc/xdg/autostart -DKDE_INSTALL_QTPLUGINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib/qt-5.15.17/plugins -DKDE_INSTALL_PLUGINDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib/qt-5.15.17/plugins -DKDE_INSTALL_QMLDIR=/nix/store/2n2hy2al9llaaqcdx92awdq6lm6ijds3-fancontrol-gui-0.8/lib/qt-5.15.17/qml -DECM_DIR=/nix/store/6qbd89v49lva9xkhxkjcgnjmzwd92kbg-extra-cmake-modules-5.116.0/share/ECM/cmake
       > CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
       >   Compatibility with CMake < 3.5 has been removed from CMake.
       >
       >   Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
       >   to tell CMake that the project requires at least <min> but has been updated
       >   to work with policies introduced by <max> or earlier.
       >
       >   Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
       >
       > 
       > -- Configuring incomplete, errors occurred!
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.17.2-zen1, NixOS, 25.11 (Xantusia), 25.11.20251025.6a08e6b`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - nixpkgs: `/nix/store/jd3y5kfy3ywv9lc229nz8vxqbrcyv93d-source`

### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/fa/fancontrol-gui/package.nix
+++ b/pkgs/by-name/fa/fancontrol-gui/package.nix
@@ -42,6 +42,11 @@ stdenv.mkDerivation rec {
     ./version.patch
   ];
 
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace-fail "cmake_minimum_required(VERSION 3.0.2)" "cmake_minimum_required(VERSION 3.10)"
+  '';
+
   passthru = {
     tests.version = testers.testVersion {
       package = fancontrol-gui;

